### ISSUE TITLE
Map generation from Json

### ISSUE THREAD

**Issue by Wasorma** [blacha/diablo2#150]
Hi,

Thanks for posting this repo.

I have the map package working, however am unable to tell how to convert the output json into a png like you show in the readme images.

**Comment by blacha** [comment]
I have built two options for map visualisations from this JSON map object.

1. Web based using maplibre/HTML canvas https://github.com/blacha/diablo2/blob/master/packages/memory/src/session.ts#L62
2. Older versions using nodejs canvas object to render to a PNG This is from my old `pimap` repo that I haven't ported across to the new code base yet. https://github.com/blacha/pimap/blob/0588544b9e37f49bbd7ae7e7d4e436599c5e498a/src/mapserver/map.image.ts

**Comment by Wasorma** [comment]
Thanks for the response

Regarding method #1, I see maplibre code in the map package, do you mind giving an example of how to view it?

**Comment by mgalos999** [comment]
Yeah I was thinking I'll make it a package, where you send it JSON blob and a location to dump the SVGs. Then just add it as a dependency to your map server and add a new endpoint to trigger it.
